containers/matomo/config.ini.php

```diff
@@ -151,9 +152,13 @@
 PluginsInstalled[] = "UserId"
 PluginsInstalled[] = "CustomPiwikJs"
 PluginsInstalled[] = "WAIMatomoTheme"
+PluginsInstalled[] = "UsersManagerOnlyAdmin"
```

Comment:
Questa configurazione dovrebbe essere gestita come per gli altri plugin (es. TrackerHostname o LoginFilterIp) e non scritta direttamente in `config.ini.php`.